src/components/facets/FacetRow.tsx

```diff
@@ -9,48 +10,96 @@ export interface IFacetRowProps extends React.ClassAttributes<FacetRow> {
     onToggleFacet: (facetRow: IFacet) => void;
     isChecked: boolean;
     maxTooltipLabelLength?: number;
+    enabledExclude?: boolean;
 }
 
 export class FacetRow extends React.Component<IFacetRowProps, any> {
     static defaultProps: Partial<IFacetRowProps> = {
         maxTooltipLabelLength: 25,
     };
 
-    render() {
-        const maxCalculatedNameLength = this.props.facetRow.count
-            ? this.props.maxTooltipLabelLength - this.props.facetRow.count.length
-            : this.props.maxTooltipLabelLength;
-        const label: JSX.Element = this.props.facetRow.formattedName.length > maxCalculatedNameLength
-            ? <Tooltip title={this.props.facetRow.tooltipLabel || this.props.facetRow.formattedName} placement='top' className='inline-block full-content-x'>
-                {this.props.facetRow.formattedName}
-            </Tooltip>
-            : <span>{this.props.facetRow.formattedName}</span>;
-        const count: JSX.Element = !_.isUndefined(this.props.facetRow.count)
-            ? <span className='facet-value-count'>{this.props.facetRow.count}</span>
-            : null;
+    get isExclude(): boolean {
+        return !!this.props.enabledExclude && !!this.props.facetRow.isExclude;
+    }
 
+    render() {
+        const className: string = `facet-value facet-selectable ${!!this.props.enabledExclude ? 'facet-exclude' : ''}`;

```

Comment:
You could use the `classNames` util here (`import * as classNames from 'classnames'`):

```
const className: string = classNames(
    'facet-value',
    'facet-selectable',
    {'facet-exclude': this.props.enabledExclude}
);
```